lib/client/clock-client.js

```diff
@@ -42,7 +59,7 @@ client.render = function render (xhr) {
   $('#bgnow').html(displayValue);
 
   // Insert the trend arrow.
-  $('#arrow').attr('src', '/images/' + rec.direction + '.svg');
+  $('#arrow').attr('src', '/images/' + (!rec.direction || rec.direction === 'NOT COMPUTABLE' ? 'NONE' : rec.direction) + '.svg');
```

Comment:
If calculation cannot be done, so invalid icon was shown. This will fix the icon.